backend/corpora/common/corpora_orm.py

```diff
@@ -228,7 +229,8 @@ class DbCollection(Base, AuditMixin, TimestampMixin):
     # the tablename is "project" instead of "collection" to avoid migrating the database
     __tablename__ = "project"
 
-    visibility = Column(Enum(CollectionVisibility), primary_key=True, nullable=False)
+    id = Column(String, primary_key=True, default=generate_uuid)
```

Comment:
rm, this key is defined in Base.
```suggestion
```

Reply:
have to have here because otherwise Python sees `id` as the builtin function, not an attribute for this class. Super annoying. But I couldn't come up with another solution.

Reply:
I should add a comment for this line though